javascript/chronometer.js

```diff
@@ -1,34 +1,48 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
   }
 
   start(callback) {
-    // ... your code goes here
+    this.intervalId = setInterval(() => {
+  this.currentTime++}, 1000);
   }
 
+
   getMinutes() {
-    // ... your code goes here
+
+    return Math.floor(this.currentTime/60);
+    
   }
 
   getSeconds() {
-    // ... your code goes here
+    
+    return Math.floor(this.currentTime % 60);
```

Comment:
The modulo operator already gives you a full integer so there is no need for Math.floor()